osu.Game/Rulesets/Edit/HitObjectComposer.cs

```diff
@@ -110,6 +110,13 @@ private void load(IBindableBeatmap beatmap, IFrameBasedClock framedClock)
             toolboxCollection.Items[0].Select();
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            rulesetContainer.Playfield.DisplayJudgements.Value = false;
```

Comment:
Eventually i'd hope this is set directly on `RulesetContainer` rather than going this far down the chain, but I guess this is okay for now.

Reply:
I considered that too, not sure if `RulesetContainer` is fit for it though, since it's more of a "general component that houses everything"... Worth a discussion